recipe/gen_patch_json.py

```diff
@@ -994,6 +994,17 @@ def _gen_new_index(repodata, subdir):
             new_depends = record.get("depends", [])
             new_depends.append("libgcc-ng <=9.3.0")
             record["depends"] = new_depends
+        
+        # setutools started raising a warning when using `LooseVersion` from distutils
+        # since packages don't tend to pin setuptools, this raises warnings in old versions
+        # https://github.com/conda-forge/conda-forge.github.io/issues/1575
+        if (
+            record_name in ["pandas", "distributed", "dask"]
+            and record.get("timestamp", 0) < 1640101398654  # 2021-12-21
+        ):
+            new_depends = record.get("depends", [])
```

Comment:
I had not realized that the distributed feedstock does explicitly include setuptools. I think this change should handle that:

```suggestion
            new_depends = record.get("depends", [])
            if "setuptools" in new_depends:
                new_depends.remove("setuptools")
```

Reply:
Don't do this. People have run deps on setuptools for lots of reasons.

Reply:
ohhhh nvm you are removing the old one.

Reply:
Normally what we do is find the constraint and replace it

Reply:
I am now finding the constraint and replacing as suggested.